modules/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client",
   "author": "Aragon Association",
-  "version": "0.17.0-alpha",
+  "version": "0.17.1-alpha",
```

Comment:
it's ok now, but the very next version adding features I would already label it as `beta`. Feel free to resolve